doc/reference_accessor.xml

```diff
@@ -1503,6 +1500,103 @@ FROM (SELECT ST_BuildArea(
 	  </refsection>
 	</refentry>
 
+    <refentry xml:id="ST_NumCurves">
+      <refnamediv>
+        <refname>ST_NumCurves</refname>
+        <refpurpose>Return the number of component curves in a CompoundCurve.</refpurpose>
+      </refnamediv>
+
+      <refsynopsisdiv>
+        <funcsynopsis>
+          <funcprototype>
+            <funcdef>integer <function>ST_NumCurves</function></funcdef>
+            <paramdef><type>geometry </type> <parameter>a_compoundcurve</parameter></paramdef>
+          </funcprototype>
+        </funcsynopsis>
+      </refsynopsisdiv>
+
+      <refsection>
+        <title>Description</title>
+        <para>Return the number of component curves in a CompoundCurve, zero for an empty CompoundCurve, or NULL for a non-CompoundCurve input. </para>
+        <para>&sqlmm_compliant; SQL-MM 3: 8.2.6, 8.3.5</para>
+        <para>&Z_support;</para>
+      </refsection>
+
+      <refsection>
+        <title>Examples</title>
+
+        <programlisting>
+-- Returns 3
+SELECT ST_NumCurves('COMPOUNDCURVE(
+    LINESTRING(2 2, 2.5 2.5),
```

Comment:
My understanding of https://github.com/postgis/postgis/blob/8ed84517a9b86c86724504d1b6a0f0c1ccd86cf6/doc/bnf-wkt.txt is that the explicit LINESTRING() keyword within a COMPOUNDCURVE is not allowed, and a PostGIS, OGR, etc. extension. It should probably not be advertized here (and below)